lesson05/assignment/index.html

```diff
@@ -1,15 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 05 - CSS Grid</title>
-    </head>
-    <body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Assignment 05 - CSS Grid</title>
+</head>
+<body>
+  <div class="grid-container">
+    <div class="header">
+      <header>
+        <div class="logo">
+          <img src="image/yoga-chakra.jpg" id ="yoga-chakra" alt="man sitting in ayogic posture"></div>
+          <h1>Yoga</h1>
+        </header>
+      </div>
+      <div class="nav-bar">
+        <nav>
```

Comment:
You could probably apply the nav-bar CSS directly to your <nav> element, then you wouldn't have to nest it in the div.